docs/cloud/features/scheduler/airflow.md

```diff
@@ -0,0 +1,96 @@
+# Airflow
+To integrate with Airflow, add the following library to your Airflow runtime environment:
+
+```
+$ pip install tobiko-cloud-scheduler-facade[airflow]
+```
+
+You also need to add an Airflow [connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#creating-a-connection-with-the-ui) containing your Tobiko Cloud credentials:
+
+```
+id: tobiko_cloud
+type: HTTP
+host: https://tobiko/cloud/base/url
+password: <tobiko cloud API token>
+```
+
+!!! info
+
+    Note that you can name it whatever you want, you just need to remember the name because it's used for the `conn_id` parameter below.
+
+After setting up the connection, you can then create a DAG with the following content:
+```
+from tobikodata.scheduler_facades.airflow import SQLMeshEnterpriseAirflow
+
+tobiko_cloud = SQLMeshEnterpriseAirflow(conn_id="tobiko_cloud")
+
+first_task, last_task, dag = tobiko_cloud.create_cadence_dag(environment="prod")
+```
+
+This is all that's required to integrate with Tobiko Cloud!
+
+Once this DAG is loaded in Airflow, it will be populated with your SQLMesh model structure for the specified `environment` and will be automatically configured to trigger at the same time as the next Cloud Scheduler run.
```

Comment:
add a section on how to debug with this airflow integration by going to the logs tab, clicking the link, and how it shows the debugger view. Include screenshots